content/logs/processing/attributes_naming_convention.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Attributes Naming Convention
+kind: documentation
+description: "Parse your logs using the Grok Processor"
+aliases:
+    - logs/parsing/
+further_reading:
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/faq/log-parsing-best-practice"
+  tag: "FAQ"
+  text: Log Parsing - Best Practice
+- link: "logs/logging_without_limits"
+  tag: "Documentation"
+  text: Control the volume of logs indexed by Datadog
+---
+
+## Tl;Dr
+
+One may ask, why do we need a naming convention for Log attributes?
+
+* Various technologies tend to use different attribute names for the same meaning.
+* This usually generates too many attributes which can lead to: user confusion and un-ability to correlate logs across sources.
+* Datadog Integrations rely on the following attribute naming convention.
+* We advise you to use the following reference when you parse your custom formats & sources in order to leverage fully all functionalities of your Datadog application.
```

Comment:
Use the following reference when parsing custom formats and sources in order to fully leverage all of Datadog's functionalities.